app/dev_mode/views.py

```diff
@@ -38,11 +38,26 @@ def dev_mode():
         return_by = form.get("return_by")
         employment_date = form.get("employment_date")
         region_code = form.get("region_code")
+        language_code = form.get("language_code")
```

Comment:
The longer term plan wasn't to duplicate the schema like this but to hold the strings separately for translations and use the schema for structure (future target)